package.json

```diff
@@ -24,28 +24,32 @@
     "index.js"
   ],
   "keywords": [
+    "256",
     "ansi",
-    "styles",
+    "cli",
```

Comment:
`cli` and `command-line` makes me think this package is more something like `chalk-cli`.